content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md

```diff
@@ -4,8 +4,8 @@ api_metadata:
   import: "k8s.io/apimachinery/pkg/apis/meta/v1"
   kind: "DeleteOptions"
 content_type: "api_reference"
-description: "删除 API 对象时可能会提供删除选项。"
-title: "删除选项"
+description: "删除 API 对象时可能会提供 DeleteOptions。"
```

Comment:
```suggestion
description: "删除 API 对象时可以提供 DeleteOptions。"
```

Reply:
Hi @tengqm , could you put an reviewer's eye here again. If no more comments, could we approve and merge this small request, so I can submit the next localized page :)